conductr_cli/exceptions.py

```diff
@@ -266,3 +266,11 @@ def __init__(self, message):
 
     def __str__(self):
         return repr(self.message)
+
+
+class ConductBackUpError(Exception):
+    def __init__(self, messages):
+        self.messages = messages
+
+    def __str(self):
+        return repr(os.linesep.join(self.messages))
```

Comment:
I think this line indicates that `self.messages` is an array of string, but in the code I noticed that `ConductBackUpError` is created with a string instance instead.

Would you be able to confirm whether `self.messages` is an array of string or a string instance?

If it's an array of string, then all the raise for `ConductBackupError` must be updated, i.e. `raise ConductBackupError(['My error message'])`

If it's a string instance, the `__str__` has to be:

```
    def __str__(self):
          return repr(self.message)
```